Symfony/CS/ConfigInterface.php

```diff
@@ -59,6 +59,8 @@ public function getFixers();
      * Sets the root directory of the project.
      *
      * @param string $dir The project root directory
+     *
+     * @return Config The same instance
```

Comment:
returning `$this`, not interface, makes sense on IDE typehinting, This is example:
```php
interface SomeI
{
    /**
     * @return $this
     */
    public function doSomething();
}

class SomeC implements SomeI
{
    public function method()
    {
        // code...
    }

    /**
     * {@inheritDoc}
     */
    public function doSomething()
    {
        // implementation
    }
}

$a = new SomeC();
$a->doSomething()-> // Here method() is typehinted because doSomething() annotated with @return $this.
```